tests/Functional/ContentConsultationRenderTest.php

```diff
@@ -29,8 +30,12 @@ class ContentConsultationRenderTest extends ContentRenderTestBase {
     'system',
     'path',
     'field_group',
+    'oe_content_entity',
+    'oe_content_sub_entity_document_reference',
     'oe_theme_helper',
     'oe_theme_content_consultation',
+    'oe_theme_content_publication',
+    'oe_theme_content_document_reference',
```

Comment:
oe_theme_content_document_reference should be moved to oe_theme_content_consultation dependency